src/components/TechTalkGroups/TechTalkGroups.js

```diff
@@ -43,37 +43,37 @@ const TechTalks = ({techTalks}) => {
   return (
     <Box m="auto" maxWidth={theme.breakpoints.values.lg}>
       <Typography variant="h1" color="textPrimary">
-        Tech Talks
+        Tech Talk Groups
       </Typography>
-      {techTalks
+      {techTalkGroups
         .slice((page - 1) * ITEMS_PER_PAGE, page * ITEMS_PER_PAGE)
-        .map((techTalk) => (
+        .map((techTalkGroup) => (
           <Box
             display="flex"
             flexDirection="column"
-            key={techTalk.frontmatter.title}
+            key={techTalkGroup.frontmatter.title}
```

Comment:
do groups have ids?  if so, those could be used instead as title's could be non-unique.  Maybe `slug` is more appropriate here?

Reply:
Updated!